RecoDataProducts/inc/CrvCoincidenceCluster.hh

```diff
@@ -33,6 +35,10 @@ namespace mu2e
     float                                       GetSlope() const         {return _slope;}
     const std::vector<int>                     &GetLayers() const        {return _layers;}
 
+    const std::vector<size_t>                  &GetSideHits() const      {return _sideHits;}
```

Comment:
SiPMs 0,2 are always on the negative sides of a counter, i.e. at -x, -y, or -z depending on the orientation. SiPMs 1 and 3 are at the positive side.

Reply:
OK thanks.  

Eventually we need methods to translate from end to sipm, and methods to provide the (nominal) direction vector of a bar, given its SectorType.  Maybe something like Hep3Vector barDirection(int CrvSectorType),  Maybe in CosmicRayShield  or CrvHelpler.  The point is downstream code will need those, and I want them defined in a single, central place.  Implementing these would be easier if barend, sipmnumber, crvSectorType, etc, were enums or classes instead of simple ints, so that they could be distinguished in an interface.  Addressing those is beyond the scope of this PR, so I will add it as an issue.